Makefile

```diff
@@ -11,7 +11,11 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
-.PHONY: all gobuild static docker release certs test clean netkitten test-registry run-functional-tests gremlin benchmark-test gogenerate run-integ-tests image-cleanup-test-images
+PAUSE_CONTAINER_IMAGE = "amazon/amazon-ecs-pause"
+PAUSE_CONTAINER_TAG = "0.1.0"
```

Comment:
The LoadImage API uses [LoadImageOptions](https://github.com/fsouza/go-dockerclient/blob/6936859b3576e8f5abd069c0edbcd454133632a2/image.go#L342) and there isn't an explicit way to expose the tag information there. 

The tag information is persisted in the pause-container tarball which is restored upon import.